src/features/metrics/components/MetricsSidebar/AllocationList.tsx

```diff
@@ -0,0 +1,31 @@
+import { useMemo } from "react";
+import { type MetricProject } from "~/utils/fetchMetrics";
+import { AllocationItem } from "./AllocationItem";
+
+const handleFormatAllocation = (p: number) => `${p * 100}%`;
+
+export function AllocationList({
+  projects,
+  sortAscending,
+}: {
+  projects: MetricProject[];
+  sortAscending: boolean;
+}) {
+  const projectsSortedByAmount = useMemo(
+    () =>
+      [...projects].sort((a, b) =>
+        sortAscending ? a.amount - b.amount : b.amount - a.amount,
+      ),
+    [projects, sortAscending],
+  );
+
+  return (
+    <>
+      {projectsSortedByAmount.map((project) => (
+        <AllocationItem key={project.id} {...project}>
+          {handleFormatAllocation(project.amount)}
```

Comment:
Something with the calculation here. Try using `project.fraction` instead.

![image](https://github.com/gitcoinco/easy-retro-pgf/assets/2961337/f2105044-6b4a-4ae8-b8f7-fadc3a51c41d)